src/main/java/com/taboola/backstage/model/media/campaigns/items/CtaType.java

```diff
@@ -21,5 +21,6 @@ public enum CtaType {
     GET_OFFER,
     TRY_NOW,
     NONE,
-    @JsonEnumDefaultValue UNSUPPORTED_BY_SDK_VALUE
+    @JsonEnumDefaultValue UNSUPPORTED_BY_SDK_VALUE,
+    SEARCH_NOW
```

Comment:
Can you move it to be above unsupported enum?

Reply:
Done